pkg/cmd/repo/create/http.go

```diff
@@ -21,14 +21,22 @@ type repoCreateInput struct {
 	HasWikiEnabled   bool `json:"hasWikiEnabled"`
 }
 
+type repoTemplateInput struct {
+	Name       string `json:"name"`
+	Visibility string `json:"visibility"`
+	OwnerID    string `json:"ownerId,omitempty"`
+
+	RepositoryID string `json:"repositoryId,omitempty"`
+	Description  string `json:"description,omitempty"`
+}
+
 // repoCreate creates a new GitHub repository
-func repoCreate(client *http.Client, hostname string, input repoCreateInput) (*api.Repository, error) {
+func repoCreate(client *http.Client, hostname string, input repoCreateInput, template ...string) (*api.Repository, error) {
```

Comment:
a nitpick, but I've never seen variadic parameters used like this in Go. If an argument is optional, the expectation is that it is passed blank (in this case, as `""`).

Reply:
Addressed in https://github.com/cli/cli/pull/1590/commits/34c37184b2b8e4465a8917f9cd7130f27aba2025